### PR TITLE
Support inverse transformation in Transformer block

### LINKED ISSUE #61: inverse option for block Transformer
There are cases that a `sklearn` `Transformer` needs to be used for inverse transforms (e.g. classifier returns prediction in label encoded format, but we want to use the original encoding, such as categorical value. )

I suggest including an `inverse` input argument in block `Transformer` to support this feature. 

Happy to submit a PR.

### PR DESCRIPTION
**Pull request by agamemnonc**
Fixes #61.

### CODE REVIEW

File: axopy/pipeline/common.py [hunk 1]
```diff
@@ -427,17 +427,24 @@ class Transformer(Block):
...
         self.transformer = transformer
+        self.inverse = inverse
 
     def process(self, data):
-        """Calls the transformer's ``transform`` method and returns the result.
+        """Calls the transformer's ``transform`` or ``inverse_transform``
+        method and returns the result.
         """
-        return self.transformer.transform(data)
+        if self.inverse is True:
```
Comment (ixjlyons) [inline axopy/pipeline/common.py]: Nit: just `if self.inverse:`

**Comment by agamemnonc** [comment]
Thanks, done!

**Comment by ixjlyons** [comment]
Wow, I merged this without noticing Travis failed. It looks like it's unrelated to this PR though (segfault -- almost definitely Qt related :disappointed: )

**Comment by agamemnonc** [comment]
I 've noticed that Travis fails in all PRs I have submitted even when tests pass normally. Travis reports failures only for versions 3.5 and 3.6 -- I am not sure why this is happening (btw I am using Python 3.6).

**Comment by ixjlyons** [comment]
3.5 and 3.6 are the two versions where I use pip to install things rather than conda (because PyQt5 wheels aren't available for Python <= 3.4). It is a constant struggle keeping Qt stuff working in CI.

**Comment by ixjlyons** [comment]
FYI CI is fixed up now and I'm hoping it's somewhat stable for a while.

**Pull request merged**